app/assets/stylesheets/application.scss

```diff
@@ -15,10 +15,48 @@
  *= require foundation_and_overrides
  */
 
+@import url(https://fonts.googleapis.com/css?family=Open+Sans:400,300,600,700,400italic,300italic);
+
  /************** BASIC STYLING *****************************/
 
 #content {
-  background-color: #DADADA;
+  background-color: #FFF;
+  margin-bottom: 6rem;
+  margin-top: 6rem;
+}
+
+#background {
+  display: flex;
+  flex-flow: column;
+  height: 100%;
+  background: url("http://freephotos.atguru.in/hdphotos/forest-landscape-wallpaper/forest-sun-wallpaper-landscape-nature-wallpaper-2560-1600-widescreen-1080.jpg") no-repeat center center fixed;
+  -webkit-background-size: cover;
+  -moz-background-size: cover;
+  -o-background-size: cover;
+  background-size: cover;
+  overflow: auto;
+}
+
+/************** NAVIGATION *******************************/
+
+#nav-container {
+  border-bottom: 1px solid #d9d9d9;
+  box-shadow: 2px 0 5px rgba(0, 0, 0, 0.2);
+}
+
+.dropdown {
+  border: 1px solid #d9d9d9;
+  border-top: none;
+  box-shadow: 0 4px 5px 0 rgba(0, 0, 0, 0.2);
+}
+
+.dropdown li > a {
+  border-bottom: 1px solid #d9d9d9;
```

Comment:
Color literals like `#d9d9d9` should only be used in variable declarations; they should be referred to via variable everywhere else.